README.md

```diff
@@ -8,9 +8,19 @@ Setup:
     virtualenv venv
     source venv/bin/activate
     pip install -r requirements.txt
+    python generate_secret.py   # follow the instructions when you run this
 
 Running:
 
-    python run.py
+    python run.py   # flask automatically reloads on changes
 
 Go to  `localhost:5000`
+
+Wiki
+---
+Built with [Flask](http://flask.pocoo.org/) and [pdf.js](https://github.com/mozilla/pdf.js)
+
+Directory structure:
+- `\instance` stores all data(eg. config, uploaded files) about current running server
```

Comment:
Aren't we trying to avoid persistence? Per our conversation yesterday? Curious to hear your thoughts! :)